.github/workflows/build-images-releases.yaml

```diff
@@ -3,8 +3,8 @@ name: Image Release Build
 on:
   push:
     tags:
-      - v[0-9]+.[0-9]+.[0-9]+
-      - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+      - v1.12.[0-9]+
+      - v1.12.[0-9]+-rc[0-9]+
```

Comment:
I'm always a bit confused which of these should be configured on master vs. the older branch. Doesn't the master branch version of this workflow already cover the 1.12 releases?

Reply:
I got confused about this as well but then I remembered that this GH workflow is executed for tags pushed **into** this branch. That's why we have a specific GH workflow for each branch.